core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -972,6 +974,10 @@ boolean goToBookmarks() {
     return true;
   }
 
+  @Override public void onToggledFullscreen(@NotNull Boolean isFullScreen) {
+
+  }
+
```

Comment:
@macgills Is this ok? or should I make `CustomReaderFragment` to abstract?

Reply:
I'd add a comment to the empty method `// does nothing because custom doesn't have a nav bar` or something to that effect as a reflex from years of static analysis warnings

Reply:
@macgills should I just add a comment in the empty function?

Reply:
yeah